clang/lib/Lex/Preprocessor.cpp

```diff
@@ -862,6 +862,15 @@ bool Preprocessor::HandleIdentifier(Token &Identifier) {
     ModuleImportExpectsIdentifier = true;
     CurLexerCallback = CLK_LexAfterModuleImport;
   }
+
+  if ((II.isModulesDeclaration() || Identifier.is(tok::kw_module)) &&
+      !InMacroArgs && !DisableMacroExpansion &&
+      (getLangOpts().Modules || getLangOpts().DebuggerSupport) &&
```

Comment:
Can you explain these two options? I would expect that to be `CPlusPlusModules`

Reply:
Good catch! This is definitely a mistake. It should be `CPlusPlusModules `. And the `DebuggerSupport` was used for debug mode, I followed what `import` doing.